GLAA.Web/Views/PublicRegister/Index.cshtml

```diff
@@ -0,0 +1,84 @@
+﻿@model GLAA.ViewModels.PublicRegister.PublicRegisterLicenceListViewModel
+
+@{
+    ViewBag.Title = "GLAA - Public Register";
```

Comment:
Not using the `Title` from the viewmodel?

Reply:
Removed